xinference/deploy/docker/docker-compose-distributed.yml

```diff
@@ -0,0 +1,60 @@
+version: '3.8'
+
+services:
+  xinference: &xinference
+    image: xprobe/xinference:latest
+    deploy:
+      resources:
+        reservations:
+          devices:
+            - capabilities: [gpu]
+              driver: nvidia
+              count: all
+#    volumes:
+#      # Replace <xinference_home> with your xinference home path on the host machine
+#      - <xinference_home>:/root/.xinference
+#      # Replace <huggingface_cache_dir> with your huggingface cache path, default is
+#      # <home_path>/.cache/huggingface
+#      - <huggingface_cache_dir>:/root/.cache/huggingface
+#      # If models are downloaded from modelscope, replace <huggingface_cache_dir> with
+#      # your modelscope cache path, default is <home_path>/.cache/modelscope
+#      - <modelscope_cache_dir>:/root/.cache/modelscope
+#    environment:
+#      # add envs here. Here's an example, if you want to download model from modelscope
+#      - XINFERENCE_MODEL_SRC=modelscope
+
+  xinference-supervisor:
+    <<: *xinference
+    ports:
+      - "9997:9997"
+      - "9999:9999"
+    command: xinference-supervisor --host xinference-supervisor --port 9997 --supervisor-port 9999
+    restart: always
+    healthcheck:
+      test: curl --fail http://xinference-supervisor:9997/status || exit 1
+      interval: 60s
```

Comment:
Recommand to reduce this to 5s. Otherwise the workers will wait longer to start.